src/main/resources/templates/start.html

```diff
@@ -29,12 +29,12 @@ <h2>Collections</h2>
     </div>
     <!-- Items of a selected collection section-->
     <div class="collection-div">
-        <div class="container item-div">
+        <div class="container pt-3 pb-3">
             <div class="row row-cols-auto">
                 <h2>Browse Collection</h2>
-                <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addItemModal">Add Item</button>
+                <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#addItemModal">Add Item</button>
```

Comment:
The background and foreground here were both blue, which can make it hard to see the button. I changed it to a secondary color that matched all the other buttons on the page. We want to make sure there is plenty of contrast between the foreground and background elements.

Citation: https://www.w3.org/WAI/tips/designing/#provide-sufficient-contrast-between-foreground-and-background